diamond_agent/tests/resources/blueprint/restart_diamond_script.yaml

```diff
@@ -0,0 +1,52 @@
+tosca_definitions_version: cloudify_dsl_1_2
+
+imports:
+    - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-manager/master/resources/rest-service/cloudify/types/types.yaml
```

Comment:
(after discussing this on slack we decided to stick with the rawgithub link for now)